pkgs/development/python-modules/unicode-slugify/default.nix

```diff
@@ -18,16 +18,20 @@ buildPythonPackage rec {
     sha256 = "25f424258317e4cb41093e2953374b3af1f23097297664731cdb3ae46f6bd6c3";
   };
 
+  patches = [ ./use_pytest_instead_of_nose.patch ];
+
   propagatedBuildInputs = [
     six
     unidecode
   ];
 
   nativeCheckInputs = [
-    nose
+    pytestCheckHook
     unittestCheckHook
   ];
```

Comment:
Do we really need both of these? Ideally we’d just use `pytestCheckHook` for the whole thing.